logo/make_logo.py

```diff
@@ -12,6 +16,19 @@ def __add__(self, other: 'Point') -> 'Point':
 		return Point(self.x + other.x, self.y + other.y)
 
 
+def font_subset():
+	font = TTFont('font/Minecrafter.Reg.ttf')
+	subsetter = Subsetter()
+	subsetter.populate(text=''.join(set('MCDaemonReforged')))
+	subsetter.subset(font)
+	font.flavor = 'woff'
+	font.save('font/Minecrafter.min.woff')
```

Comment:
Is it possible to keep the `.min.woff` content in memory instead of writing it to a file?